File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/StatusApplicantDTO.java
package com.example.befindingjob.dto;

import com.example.befindingjob.entity.enumm.ApplicationStatus;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Data
@Getter
@Setter
public class StatusApplicantDTO {
    private int applicationId;
    private ApplicationStatus status;
    private String message;
    private LocalDateTime interviewDate;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/StatusApplicantDTO.kt
package com.example.jobhub.dto

import com.example.jobhub.entity.enumm.ApplicationStatus
import java.time.LocalDateTime

data class StatusApplicantDTO (
    var applicationId: Int? = 0,
    var status: ApplicationStatus? = ApplicationStatus.APPLIED,
    var message: String? = "",
    var interviewDate: LocalDateTime? = LocalDateTime.now()
)